bdk-ffi/justfile

```diff
@@ -3,3 +3,7 @@ build:
 
 test:
   cargo test --lib
+
+check:
```

Comment:
interested to see if I can make my hands break their muscle memory and `just check` now 🤞